portfolio/src/main/webapp/WEB-INF/styles/reset.css

```diff
@@ -0,0 +1,11 @@
+:root {font : 20px/1.2 "Roboto", sans-serif;}
+
+body,
+body * {
+    margin: unset;
+    box-sizing: unset;
+    padding: unset;
+    font-size: unset;
+    color: unset;
+    text-decoration: unset;
+}
```

Comment:
It's generally a good idea (for a lot of tools) to end files with a newline (but not a blank, empty line). The red "slashed circle" plus "carriage return" indicator on the diff is telling you there is no trailing newline. Please add one.

(A word about my review style: I'll often make comments -- particularly about general principles -- that apply to multiple places in the PR. I generally only make the comment once, with the hope that you'll go through and apply the comment/principle as appropriate throughout the PR. This is one of those comments :))

Reply:
Okay, I'll take note to your style.